layers/+lang/emacs-lisp/README.org

```diff
@@ -229,3 +230,22 @@ In =debugger-mode= (=Debugger= is displayed in major mode segment of the mode
 line)
 
 TODO
+
+** Refactoring with emr
+
+| Key binding | Description                  |
+|-------------+------------------------------|
+| ~SPC r f e~ | implement function           |
```

Comment:
Since these are major mode specific, they go under `SPC m r` and not `SPC r`

Reply:
Fixed here and in changelog, thanks.